File: src/test/java/eu/sig/training/ch03/FlagsTest.java
package eu.sig.training.ch03;

import static org.junit.Assert.assertEquals;

import java.awt.Color;
import java.util.List;

import org.junit.Test;

public class FlagsTest {

    @Test
    public void testGetFlag() {
        FlagFactory factory = new FlagFactory();

        List<Color> colors = factory.getFlagColors(Nationality.DUTCH);

        assertEquals(Color.RED, colors.get(0));
        assertEquals(Color.WHITE, colors.get(1));
        assertEquals(Color.BLUE, colors.get(2));
    }

}

File: src/test/csharp/eu/sig/training/ch03/FlagsTest.cs
using NUnit.Framework;
using System.Collections.Generic;
using System.Drawing;

namespace eu.sig.training.ch03
{
    [TestFixture]
    public class FlagsTest
    {

        [Test]
        public void testGetFlag()
        {
            FlagFactory factory = new FlagFactory();

            IList<Color> colors = factory.GetFlagColors(Nationality.DUTCH);

            Assert.AreEqual(Color.Red, colors[0]);
            Assert.AreEqual(Color.White, colors[1]);
            Assert.AreEqual(Color.Blue, colors[2]);
        }

    }
}
